instrumentation/hibernate/hibernate-common/library/hibernate-common-library.gradle

```diff
@@ -0,0 +1,9 @@
+/*
+ * Classes that are common to all versions of the Hibernate instrumentation.
+ */
+
+apply from: "$rootDir/gradle/instrumentation-library.gradle"
+
+dependencies {
+  compileOnly project(':javaagent-api')
```

Comment:
since this needs `javaagent-api`, does it make sense to keep this module as `hibernate-common-javaagent`?

Reply:
Moved back